src/app/compare/compare.component.ts

```diff
@@ -54,8 +47,8 @@ export class CompareComponent implements AfterViewInit {
   }
 
   saveDiff(leftSide: string, rightSide: string) {
-    this.diffComponent.left = leftSide;
-    this.diffComponent.right = rightSide;
-    this.diffComponent.renderDiffs();
+    // this.diffComponent.left = leftSide;
+    // this.diffComponent.right = rightSide;
+    // this.diffComponent.renderDiffs();
   }
```

Comment:
Is this function even needed if everything is commented out or are you gonna use it again at a later point?

Reply:
I'm still figuring out what its use was, its not covered by the tests so its a little harder to find out